([](Runtime::Exception&& exception) {
		Errors::fatalf("Runtime exce